docs/source/examples.rst

```diff
@@ -32,27 +32,20 @@ Basic Concepts
 
     class ManimCELogo(Scene):
         def construct(self):
+            self.camera.background_color = "#ece6e2"
             logo_green = "#87c2a5"
             logo_blue = "#525893"
             logo_red = "#e07a5f"
-            ds_m = MathTex(r"\mathbb{M}", z_index=20).scale(7)
-            ds_m.shift(2.25*LEFT + 1.5*UP)
-            circle = Circle(color=logo_green,
-                            fill_opacity=1,
-                            z_index=7)
-            square = Square(color=logo_blue,
-                            fill_opacity=1,
-                            z_index=5)
-            triangle = Triangle(color=logo_red,
-                                fill_opacity=1,
-                                z_index=3)
-            circle.shift(LEFT)
-            square.shift(UP)
-            triangle.shift(RIGHT)
-            logo = VGroup(triangle, square, circle, ds_m) # order matters
+            logo_black = "#343434"
```

Comment:
Comment (non-blocking): We should really have the logo colors in utils.Colors I think